Makefile

```diff
@@ -92,10 +95,18 @@ cluster-down:
 cluster-sync:
 	VERSION=$(VERSION) ./cluster/sync.sh
 
-test/e2e/workflow:
+cluster-sync: cluster-operator-push cluster-operator-install
+
```

Comment:
Duplicate `cluster-sync:` entry? Is this something wanted & if so does the double entry improves readability?

Reply:
It was requested earlier in another PR to keep `cluster-sync` since it is well known in kubevirt projects. This target does what would a new user who saw other projects expect from sync IMO

Reply:
Yes, but there are two entries now:
```
cluster-sync:
	VERSION=$(VERSION) ./cluster/sync.sh

cluster-sync: cluster-operator-push cluster-operator-install
```

Is this something we want?

Reply:
ahhh, no, i'm just blind. thanks